manifests/machineconfigcontroller/clusterrole.yaml

```diff
@@ -16,8 +16,11 @@ rules:
   resources: ["images", "clusterversions", "featuregates", "nodes", "nodes/status"]
   verbs: ["*"]
 - apiGroups: ["config.openshift.io"]
-  resources: ["schedulers", "apiservers", "infrastructures"]
+  resources: ["schedulers", "apiservers", "infrastructures", "imagetagmirrorsets"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["config.openshift.io"]
+  resources: ["imagedigestmirrorsets"]
+  verbs: ["create", "delete", "get", "list", "watch"]
```

Comment:
Is the write access still necessary? I thought the current plan is that the conversion is going to happen outside of MCO, maybe via https://github.com/openshift/oc/pull/1238 .

Reply:
oh, no. I forget to cleanup this when dropping the imagedigestmirrorset objects creat.